source/features/improve-shortcut-help.tsx

```diff
@@ -45,8 +45,8 @@ const observer = new MutationObserver(([{target}]) => {
 });
 
 function init(): void {
-	document.addEventListener('keypress', ({key}) => {
-		if (key === '?') {
+	document.addEventListener('keypress', ({key, target}) => {
+		if (key === '?' && (target as HTMLTextAreaElement).type !== 'textarea') {
```

Comment:
I think this isn't safe enough, the target could also be `input`. GitHub has other conditions this might not meet.

To avoid the error specifically, this could be:
```suggestion
		if (key !== '?' || target instanceof HTMLTextAreaElement || target instanceof HTMLInputElement) {
			return;
		}
```

and then

```js
const modal = select('body > details > details-dialog');
if (modal) {
 			observer.observe(modal, {childList: true});
}
```

Reply:
🤔

Reply:
ohh... but it works? Im confused

Reply:
The intent of this listener is to catch the opening of `details-dialog`. Just because the key `?` was pressed outside fields, it doesn't mean that GitHub opened it. They have more conditions.

This is what I meant with:

> I think this isn't safe enough, [...] GitHub has other conditions this might not meet.